openedx/core/djangoapps/waffle_utils/tests/test_init.py

```diff
@@ -12,8 +12,7 @@
 from opaque_keys.edx.keys import CourseKey
 from waffle.testutils import override_flag
 
-from openedx.core.djangoapps.waffle_utils import CourseWaffleFlag as LegacyCourseWaffleFlag
-from openedx.core.djangoapps.waffle_utils.__future__ import FutureCourseWaffleFlag as CourseWaffleFlag
```

Comment:
To support ORA and ensure this works as needed, we should now import this as `FutureCourseWaffleFlag` without renaming it, and restore `test_legacy_course_waffle_flag`, but rename it to `test_future_course_waffle_flag`. I know I am being very cautious here, so not a big deal. But, it is simple enough to do this...

Reply:
Done